### DISCUSSION TITLE
The maximum and minimum achievable rank display

### DISCUSSION THREAD

**Discussion by normalid-awa** [ppy/osu#28591] [Ideas]
Even though the new Osu! update has been implementing the rank display update but I think it's good to implement the max. and min. achievable rank display. Just like how the accuracy did. 

### It allows players to actually see what rank they can get if they mess up at the following or what rank they can achieve if they get all perfectly.

![image](https://github.com/ppy/osu/assets/52519933/8f3c87e3-d60b-4031-bc4b-6c5b484aecb9)
from pr #28614

**Comment by k3tzchen** [comment]
Would love to see this. My Setup is build on top of "maximum achievable" making it only change when i make a mistake. The current Rank display is just too chaotic for this atm. Same goes for the PP Counter, which should get a maximum and minimum achievable setting
